.github/workflows/scorecard.yml

```
@@ -0,0 +1,73 @@
+# This workflow uses actions that are not certified by GitHub. They are provided
+# by a third-party and are governed by separate terms of service, privacy
+# policy, and support documentation.
+
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '20 3 * * 0'
+  push:
+    branches: [ "main" ]
```

Can we run the scorecard on `feat-**` branches, or is there a restriction?

```suggestion
    branches:
      - "main"
      - "feat-**"
```

Btw. can we prefix the branch wit `feat-`?